src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEModuleSymbol.cs

```diff
@@ -116,6 +116,16 @@ internal enum RefSafetyRulesAttributeVersion
 
         private RefSafetyRulesAttributeVersion _lazyRefSafetyRulesAttributeVersion;
 
+        internal enum MemorySafetyRulesAttributeVersion
+        {
+            Uninitialized = 0,
+            NoAttribute,
+            Updated,
```

Comment:
Consider renaming to `Version15` (an absolute value vs. a relative "updated")

Reply:
It is still an open question what the version will be. Can add a PROTOTYPE comment though.